js/app.js

```diff
@@ -2,6 +2,7 @@
 $(document).ready(function() {
 
 // adding listeners to call the imageApper function for each square
+    // nice ...but could do this on the box class as a forEach ...
```

Comment:
Ignore `foreEach` comment. With jQuery it's even simpler....

`$('.box').on('click', imageAppear)`